packages/odt2txt/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/dstosberg/odt2txt
+TERMUX_PKG_DESCRIPTION="simple converter from OpenDocument Text to plain text."
+TERMUX_PKG_VERSION=0.5
+TERMUX_PKG_REVISION=1
```

Comment:
Don't need a revision for first version.

Reply:
Roger https://github.com/termux/termux-packages/pull/3433/commits/1a5b7fbaa7517efec2696bc9a16db0daeb4c2b13